### PR TITLE
#153114596 Notification for jobs running too fast

### PR DESCRIPTION
**Pull request by ivanatu**
#### What does this PR do?
Have notifications for the jobs running too fast
#### Description of Task to be completed?
- User gets notified by email when a job runs too fast
- An icon is displayed on the checks to indicate jobs running fast.
#### How should this be manually tested?
- WIth all the installations available, please run the following commands:
  . python manage.py sendalerts or ./manage.py sendalerts
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
[#153114596](https://www.pivotaltracker.com/story/show/153114596)
#### Screenshots (if appropriate)
screen shot 2017-12-20 at 22 54 00
screen shot 2017-12-20 at 22 54 17

#### Questions:
- None

### CODE REVIEW

**Review by jonathankamau — CHANGES_REQUESTED** [review]
The PT ID under `What are the relevant pivotal tracker stories?` is not a link. Please fix.

**Review by jonathankamau — CHANGES_REQUESTED** [review]
You have conflicts on your PR that need fixing.

File: hc/api/models.py [hunk 1]
```diff
@@ -14,252 +14,262 @@
...
 
 class Ping(models.Model):
-    n = models.IntegerField(null=True)
-    owner = models.ForeignKey(Check)
-    created = models.DateTimeField(auto_now_add=True)
-    scheme = models.CharField(max_length=10, default="http")
-    remote_addr = models.GenericIPAddressField(blank=True, null=True)
-    method = models.CharField(max_length=10, blank=True)
-    ua = models.CharField(max_length=200, blank=True)
+	n = models.IntegerField(null=True)
```
Comment (jonathankamau) [inline hc/api/models.py:156]: Why have you added extra tabs on lines in this file?

Comment (ivanatu) [reply hc/api/models.py:156]: we didn't . That tab was already in the file

Comment (jonathankamau) [reply hc/api/models.py:156]: Ok

**Review by jonathankamau — CHANGES_REQUESTED** [review]
I just merged a PR. Please fix the conflicts that have resulted from that then let me know.

**Pull request merged**